example/src/blocks/rich-text-character-limit/index.js

```diff
@@ -1,7 +1,7 @@
 import { registerBlockType } from '@wordpress/blocks';
 import { __ } from '@wordpress/i18n';
 import { useBlockProps } from '@wordpress/block-editor';
-import { RichTextCharacterLimit } from '@10up/block-components';
+import { RichTextCharacterLimit } from '../../../../components/rich-text-character-limit';
```

Comment:
@Sidsector9 What's the reason for this change here? I don't think we should use the relative file path here.

Reply:
Now that you ask, I can't recollect why I did this 🤔 

I fixed it in [45fc61f](https://github.com/10up/block-components/pull/158/commits/45fc61fc63b001b59d8ad2cbd05ae589632c85fe)

FYI, as per the design, there's work pending on the green icon. A tick inside the circle is missing which needs to be added.